src/compas/rpc/services/default.py

```diff
@@ -15,13 +15,22 @@ class DefaultService(Service):
 
 if __name__ == '__main__':
 
+    import sys
     import threading
```

Comment:
not used anymore

Reply:
it is used, I pass the port param to the default service, so that launching it matches the port specified on the other side (otherwise one would need to manually edit the default service or create an alternative service matching the port)

Reply:
no, i meant `threading`